components/Row.module.css

```diff
@@ -37,6 +37,7 @@ div.iconContainer{
 .collapsibleTableCell {
 	padding-bottom: 0;
 	padding-top: 0;
+	background-color: #f5f5f5;
```

Comment:
maybe this colour should come from the global.css file for consistency? as in `var(--colour-lightGrey)`